### ISSUE TITLE
Add "Create" button to Login page

### ISSUE THREAD

**Issue by johneastman** [johneastman/todo#256]
* Dependent on #254
* The user should be able to create an account if they don't have one.
* If the account already exists, return an error saying "username already taken".
* A backend endpoint is created to check if the user already exists.

**Comment by johneastman** [comment]
Login page removed, so this task is no longer relevant.
